background_scripts/main.js

```diff
@@ -182,6 +182,67 @@ const mkRepeatCommand = (command) => (function (request) {
   }
 });
 
+function nextZoomLevel(currentZoom, steps) {
+  const chromeLevels = [ // Chrome's default zoom levels.
```

Comment:
Let's put the comment above the variable declaration. Also, these zoom levels can be fit all on one line if you remove the trailing zeroes. The list becomes less readable, but the values are uninteresting, so putting them on separate lines is not worth the giving them 30 lines of this source file.